lib/consensus/raft/config.go

```diff
@@ -0,0 +1,319 @@
+package consensus
+
+import (
+	"time"
+
+	hraft "github.com/hashicorp/raft"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/lotus/node/config"
+)
+
+// ConfigKey is the default configuration key for holding this component's
+// configuration section.
+var configKey = "raft"
+var envConfigKey = "cluster_raft"
+
+// Configuration defaults
+var (
+	DefaultDataSubFolder        = "raft"
+	DefaultWaitForLeaderTimeout = 15 * time.Second
+	DefaultCommitRetries        = 1
+	DefaultNetworkTimeout       = 100 * time.Second
+	DefaultCommitRetryDelay     = 200 * time.Millisecond
+	DefaultBackupsRotate        = 6
+	DefaultDatastoreNamespace   = "/r" // from "/raft"
+)
+
+// Config allows to configure the Raft Consensus component for ipfs-cluster.
```

Comment:
We'll want to remove all this commented-out code before merging

Reply:
yes will do a big cleanup soon